ui/tests/acceptance/secrets/backend/kv/kv-v2-workflow-edge-cases-test.js

```diff
@@ -309,6 +319,20 @@ module('Acceptance | kv-v2 workflow | edge cases', function (hooks) {
     assert.false(codemirror().getValue().includes('*'), 'Values are not obscured on edit view');
   });
 
+  test('on enter the JSON editor cursor goes to the next line', async function (assert) {
+    // see issue here: https://github.com/hashicorp/vault/issues/27524
+    const predictedCursorPosition = JSON.stringify({ line: 0, ch: 9, sticky: null });
+    await visit(`/vault/secrets/${this.backend}/kv/create`);
+    await fillIn(FORM.inputByAttr('path'), 'json jump');
+
+    await click(FORM.toggleJson);
+    await fillIn(GENERAL.codemirrorTextarea, '{ "json":');
```

Comment:
I tried this locally and this didn't trigger the bug behavior, maybe I'm missing something?